rules/windows/defense_evasion_masquerading_renamed_autoit.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/09/01"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious AutoIt process execution was detected. Malwares written as AutoIt scripts tend to rename AutoIt main
+executable to avoid attention. Verify process details such as command line, network connections, file writes and parent
+process details as well.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "lucene"
+license = "Elastic License"
+name = "Renamed AutoIt Scripts Interpreter"
+risk_score = 47
+rule_id = "2e1e835d-01e5-48ca-b9fc-7a61f7f11902"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process AND event.type:(start OR process_started) AND
+  process.pe.original_file_name:/[aA][uU][tT][oO][iI][tT]\d\.[eE][xX][eE]/ AND
```

Comment:
`\d` - will it ever increment to a double digit version? If so, you can add `\d+` or bind it `\d{1,3}`

Reply:
current vers is 3 still adjusted just in case